internal/bundle/plain_test.go

```diff
@@ -0,0 +1,124 @@
+package bundle
+
+import (
+	"io/fs"
+	"testing/fstest"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Plain bundle v0 loader", func() {
+	It("should fail if there is no manifests directory", func() {
+		mockfs := fstest.MapFS{}
+		_, err := LoadPlainV0(mockfs)
+		Expect(err.Error()).To(ContainSubstring("file does not exist"))
```

Comment:
It looks like we're fairly consistent with using `Expect(err).To(MatchError(ContainSubstring(...))` throughout ginkgo-related tests that perform this kind of testing check.